Apps/Common/src/Services/EmailQueueService.cs

```diff
@@ -35,46 +33,28 @@ namespace HealthGateway.Common.Services
     /// </summary>
     public class EmailQueueService : IEmailQueueService
     {
-#pragma warning disable SA1310 // Disable _ in variable name
-        private const string INVITE_KEY_VARIABLE = "InviteKey";
-        private const string ACTIVATION_HOST_VARIABLE = "ActivationHost";
-        private const string ENVIRONMENT_VARIABLE = "Environment";
-        private const string CONFIG_WEB_CLIENT_SECTION = "WebClient";
-        private const string EMAIL_CONFIG_EXPIRY_SECONDS_KEY = "EmailVerificationExpirySeconds";
-        private const string EMAIL_TEMPLATE_EXPIRY_HOURS = "ExpiryHours";
-#pragma warning restore SA1310 // Restore warnings
         private readonly IEmailDelegate emailDelegate;
-        private readonly IMessagingVerificationDelegate emailInviteDelegate;
         private readonly IWebHostEnvironment environment;
         private readonly ILogger logger;
         private readonly IBackgroundJobClient jobClient;
-        private readonly IConfiguration configuration;
-        private readonly int emailVerificationExpirySeconds;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="EmailQueueService"/> class.
         /// </summary>
         /// <param name="logger">The injected logger provider.</param>
-        /// <param name="configuration">The Configuration to use.</param>
         /// <param name="jobClient">The JobScheduler queue client.</param>
         /// <param name="emailDelegate">Email delegate to be used.</param>
-        /// <param name="emailInviteDelegate">Invite email delegate to be used.</param>
         /// <param name="environment">The injected environment configuration.</param>
         public EmailQueueService(
             ILogger<EmailQueueService> logger,
-            IConfiguration configuration,
             IBackgroundJobClient jobClient,
             IEmailDelegate emailDelegate,
-            IMessagingVerificationDelegate emailInviteDelegate,
             IWebHostEnvironment environment)
         {
```

Comment:
11 errors are being reported in CommonTests/Services/EmailQueueService_Test due to this change:

```
Error	CS1729	'EmailQueueService' does not contain a constructor that takes 6 arguments
```

Reply:
Good catch. I've updated them